src/App.tsx

```diff
@@ -15,36 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
+  default = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortBy, isReversed }: { sortBy: SortType; isReversed: boolean },
```

Comment:
You can add interface instead of setting types directly 
``` 
interface SortParams {
  sortby: SortType,
  isReversed: boolean,
}

{ sortBy, isReversed }: SortParams 
```